doc/source/rest.j2

```diff
@@ -551,15 +551,134 @@ argument, and in this case the second argument passed is the value, or it.
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiples it by `4`.
+
+.. note::
+
+   `operations` can also be passed as a string, for example:
+   `"operations": "(aggregate mean (metric (metric-id aggregation) (metric-id aggregation))"`
+
+Operations between metrics can also be done, such as:
+
+{{ scenarios['get-aggregates-between-metrics']['doc'] }}
+
+Aggregation across |metrics| have different behavior depending
+on whether boundary values are set (`start` and `stop`) and if `needed_overlap`
+is set.
+
+Gnocchi expects that time series have a certain percentage of timestamps in
+common. This percent is controlled by the `needed_overlap` needed_overlap,
+which by default expects 100% overlap. If this percentage is not reached, an
+error is returned.
+
+.. note::
+
+   If `start` or `stop` boundary is not set, Gnocchi will set the missing
+   boundary to the first or last timestamp common across all series.
+
+The ability to fill in missing points from a subset of time series is supported
+by specifying a `fill` value. Valid fill values include any float or `null`. In
+the case of `null`, Gnocchi will compute the aggregation using only the
+existing points. The `fill` parameter will not backfill timestamps which contain no
+points in any of the time series. Only timestamps which have datapoints in at
+least one of the time series is returned.
+
+.. note::
+
+   A |granularity| must be specified when using the `fill` parameter.
+
+{{ scenarios['get-aggregates-by-metric-ids-fill']['doc'] }}
+
+
+
+List of supported <operations>
+------------------------------
+
+getting one or more metrics::
+
+   (metric <metric-id> <aggregation>)
+   (metric ((<metric-id> <aggregation>), (<metric-id> <aggregation>), ...))
+
+   metric-id: the id of a metric to retrieve
+   aggregation: the aggregation method to retrieve
+
+rolling window aggregation::
+
+   (rolling <aggregation method> <rolling window> (<operations>)
+
+   aggregation method: the aggregation method to use to compute the rolling window.
+                       (mean, median, std, min, max, sum, var, count)
+   rolling window: the rolling window
+
+
+aggregation across metrics::
+
+   aggregate <aggregation method> ((<operations>), (<operations>), ...))
+
+   aggregation method: the aggregation method to use to compute the aggregate between metrics
+                       (mean, median, std, min, max, sum, var, count)
+
+resampling metrics::
+
+   (resample <aggregation method> <granularity> (<operations>))
+
+   aggregation method: the aggregation method to use to compute the aggregate between metrics
+                       (mean, median, std, min, max, sum, var, count)
+
+   granularity: the granularity (ex: 1d, 60s, ...)
+
+math operations::
+
+   (<operator> <operations_or_float> <operations_or_float>)
+
+   operator: %, mod, +, add, sub, ×, mul, /, ÷, div, **, ^, pow
```

Comment:
I guess you miss some synonyms? at least `*` and `-`.

Reply:
Weird I have copy/paste... Fixed.